crud/booking_utils.py

```diff
@@ -0,0 +1,79 @@
+"""CRUD functions for Booking."""
+
+import datetime
+
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
+from crud import client_utils
+from models.booking import Booking
+from schemas.booking_schemas import BookingCreate
+
+
+def get_bookings(db: Session, skip: int = 0, limit: int = 100):
+    """Get all bookings."""
+    return db.query(Booking).offset(skip).limit(limit).all()
+
+
+def get_booking(db: Session, booking_id: int):
```

Comment:
Just FYI: Usually when you get single item, it means you should get some additional info not available in get_bookings(some comments, rates, linked info, etc), so it requires additional queries or smth. It could be as it is for now, cause you don't have any complex queries here.

Reply:
So, should I add more complex queries for bookings? I have already started creating queries that include retrieving. for example, all features of a room type or all bookings created for a specific room, etc. in `crud/misc_crud.py` file. Are there any other necessary queries to be created?